packages/devtools_app/integration_test/test/live_connection/eval_and_browse_test.dart

```diff
@@ -116,5 +124,82 @@ class _EvalTester {
     await tester.pumpAndSettle();
     await simulateKeyDownEvent(LogicalKeyboardKey.enter);
     await simulateKeyUpEvent(LogicalKeyboardKey.enter);
+    await tester.pump(longPumpDuration);
+  }
+
+  Future<void> switchToSnapshotsAndTakeOne() async {
```

Comment:
can we make this method a private helper instead of a method on EvalTester? This method doesn't really have anything to do with eval.

Reply:
It needs state and helpers of this class. So, renamed the class to make it covering wider scope.